_includes/manuals/1.12/5.8.1_database.md

```diff
@@ -0,0 +1,2 @@
+
+All Foreman instances in a cluster/group must point to the same database. This can be done during the initial installation (through flags or altering __foreman\_installer\_answers.yaml__) or by directly altering __$app_root/config/database.yaml__ (usually a symlink to the real file) and pointing the correct lifecycle to your Foreman DB, then restarting Foreman.
```

Comment:
"lifecycle" should probably be "environment", and should probably recommend changing the production env as that's the usual one used.